docs/manual/facets.md

```diff
@@ -74,6 +74,17 @@ You can modify any facet expression by clicking the “change” button to the r
 
 Facet boxes that appear in the sidebar can be resized and rearranged. You can drag and drop the title bar of each box to reorder them, and drag on the bottom bar of text facet boxes. 
 
+:::info Operations that don't respect facets
+
+Certain operations don't respect facet settings. If you perform any of the following operations while filtering your data with a facet, the operation will apply to all relevant data, columns, and/or rows in the table, not just those which you have filtered with the facet:
+
+*   Moving, Removing, Renaming, or Reordering a column
+*   Denormalizing data
```

Comment:
I wouldn't list this operation here since it's not exposed in the UI and doesn't do anything useful as far as we know. I'll open an issue to remove it.

Reply:
Okay, I removed denormalization from the list.